src/System.Drawing.Common/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,14 @@
+Compat issues with assembly System.Drawing.Common:
+CannotMakeTypeAbstract : Type 'System.Drawing.BufferedGraphicsManager' is abstract in the implementation but is not abstract in the contract.
```

Comment:
@safern can you please have a look through these differences and verify that the ref is correct?